app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -181,20 +189,37 @@ def sales_tax
   end
 
   def bulk_coop
-    # -- Prepare date parameters
-    prepare_date_params params
+    if request.format.json?
```

Comment:
Yeah I guess we have this thread where we can discuss that https://community.openfoodnetwork.org/t/road-map-for-ofns-api/931 . But in general we should put them under the `OpenFoodNetwork` namespace and a new routing like `/api/ofn/*` or something like that.